src/stories/MonacoCodeEditor.stories.js

```diff
@@ -1,27 +1,54 @@
-import React from 'react';
-import MonacoCodeEditor from '../components/MonacoCodeEditor/MonacoCodeEditor.js';
+import { withAuthentication } from "@mov-ai/mov-fe-lib-react";
+import React from "react";
+import withMock from "storybook-addon-mock";
+import MonacoCodeEditor from "../components/MonacoCodeEditor/MonacoCodeEditor.js";
 
 export default {
-  title: 'Monaco Code Editor',
+  title: "Monaco Code Editor",
   component: MonacoCodeEditor,
+  decorators: [withMock],
   argTypes: {
     theme: {
-      options: ['vs-dark', 'light'],
-      control: { type: 'radio' },
+      options: ["vs-dark", "light"],
+      control: { type: "radio" },
     },
   },
 };
 
-const Template = (args) => (
-  <div style={{ height: '90vh' }}>
-    <MonacoCodeEditor {...args} />
-  </div>
-);
+const Template = (args) => {
+  const C = (_) => (
+    <div style={{ height: "90vh" }}>
+      <MonacoCodeEditor {...args} useLanguageServer />
+    </div>
+  );
+  const A = withAuthentication(C);
+  return <A></A>;
+};
 
 export const Python = Template.bind({});
+// We need to mock authentication in storybook because proxying requests to BE is not enough (storybook sends requests through iframes, which are blocked in BE)
+Python.parameters = {
+  mockData: [
+    {
+      url: "/token-auth/",
+      method: "POST",
+      status: 200,
+      response: (_) => {
+        // MUST BE A REAL TOKEN FROM MOVAI BE
+        return {
+          refresh_token:
```

Comment:
Hi Pedro, can you omit this block of code? I recommend creating another file that exports these parameters, that would contain empty access and refresh tokens. 

The reason that I'm asking for it to be in a separate file, is so we can commit it with empty tokens and add it to the .gitignore file since this is a file that should only be changed for testing.

Reply:
fixed